css/props.css

```
@@ -0,0 +1,61 @@
+html {
+  /* fonts */
+  --wf-serif: freight-text-pro;
+  --wf-sans: freight-sans-pro;
+  --wf-code: source-code-pro;
+  --os-serif: palatino, 'palatino linotype', 'palatino lt std', 'book antiqua',
+    georgia;
+  --os-sans: 'helvetica neue', helvetica, arial;
+  --os-code: sfmono-regular, consolas, monaco;
+
+  --serif: var(--wf-serif, var(--os-serif, ui-serif)), serif;
+  --sans-serif: var(--wf-sans, var(--os-sans, ui-sans-serif)), sans-serif;
+  --code: var(--wf-code, var(--os-code, ui-monospace)), monospace, serif;
+
+  /* scale */
+  --step-n1: clamp(0.9375rem, 0.8638rem + 0.3685cqi, 1.1494rem);
+  --step-0: clamp(1.125rem, 0.9511rem + 0.8696cqi, 1.625rem);
+  --step-1: clamp(1.35rem, 1.0204rem + 1.6478cqi, 2.2975rem);
+  --step-2: clamp(1.62rem, 1.0535rem + 2.8326cqi, 3.2488rem);
+  --step-3: clamp(1.9438rem, 1.0218rem + 4.6098cqi, 4.5944rem);
+  --step-6: clamp(3.3592rem, 2.8691rem + 2.4507cqi, 4.7684rem);
+  --step-7: clamp(4.0311rem, 3.36rem + 3.3555cqi, 5.9605rem);
+  --step-8: clamp(4.8373rem, 3.9283rem + 4.5448cqi, 7.4506rem);
+  --step-9: clamp(5.8048rem, 4.5844rem + 6.1017cqi, 9.3132rem);
+  --step-10: clamp(6.9657rem, 5.3393rem + 8.1319cqi, 11.6415rem);
```

```suggestion
  --step-8: clamp(4.8373rem, 3.9283rem + 4.5448cqi, 17.4506rem);
  --step-9: clamp(5.8048rem, 4.5844rem + 6.1017cqi, 19.3132rem);
  --step-10: clamp(6.9657rem, 5.3393rem + 8.1319cqi, 19.6415rem);
```

basically, something really high at the end max number.

---

Thanks. Adjusting the max sizes and your other suggestions above made this flow much better. I added some slight adjustments to the alignment too. Ready for another look @stacyk